_sass/globals/grid.scss

```diff
@@ -177,3 +177,8 @@
   padding-left: 30px;
   padding-right: 30px;
 }
+
+.page-landing .grid-container-lg {
+  padding: 0 $sm;
+  box-sizing: content-box;
```

Comment:
This works, but I'd prefer not to override box-sizing unless it's really necessary. I posted in a separate comment what you could do instead.